cmd/snap-bootstrap/initramfs_systemd_mount_test.go

```diff
@@ -278,15 +293,21 @@ func (s *doSystemdMountSuite) TestDoSystemdMount(c *C) {
 					foundBeforeInitrdfsTarget = true
 				case strings.HasPrefix(arg, "--options="):
 					for _, opt := range strings.Split(strings.TrimPrefix(arg, "--options="), ",") {
-						switch opt {
-						case "nosuid":
+						switch opt := opt; {
+						case opt == "nosuid":
 							foundNoSuid = true
-						case "bind":
+						case opt == "bind":
 							foundBind = true
-						case "ro":
+						case opt == "ro":
 							foundReadOnly = true
-						case "private":
+						case opt == "private":
 							foundPrivate = true
+						case strings.HasPrefix(opt, "verity.hashdevice"):
```

Comment:
Should this have a trailing `=` to not be confused by things like `verity.hashdevicepotato`

Reply:
Fixed.